lib/rust/ensogl/component/breadcrumbs/src/entry.rs

```diff
@@ -263,8 +263,8 @@ impl EntryData {
         self.separator.set_size(Vector2(separator::ICON_WIDTH, size.y));
         self.ellipsis.set_size(Vector2(ellipsis::ICON_WIDTH, size.y));
         self.icon.set_size(Vector2(ICON_WIDTH, size.y));
-        self.icon.set_x(-ICON_WIDTH);
-        self.icon.set_y(-2.0);
+        self.icon.set_x(-size.x / 2.0 - 2.0);
+        self.icon.set_y(-ICON_WIDTH * 1.25 - 2.0);
```

Comment:
Offsets like `2.0` and `1.25` should probably be moved to constants.

Reply:
Done.